Firestore/core/test/firebase/firestore/auth/firebase_credentials_provider_test.mm

```diff
@@ -25,15 +31,41 @@ namespace auth {
 // Set a .plist file here to enable the test-case.
 static const char* kPlist = "";
 
+class FirebaseCredentialsProviderTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
+    app_ready_ = false;
+    absl::string_view plist(kPlist);
```

Comment:
If you declared kPlist as

```c
static NSString *const kPlist = @"";
```

You'd remove all of this conversion logic in here.